apps/projects/app/components/Content/Settings.js

```diff
@@ -114,37 +116,71 @@ SettingLabel.propTypes = {
+  onRemoveExpLevel,
   generateExpLevelHandler,
+  fundingModel,
 }) => {
+  const theme = useTheme()
   let last = expLevels[expLevels.length - 1]
   let canAdd = last && last.mul !== '' && last.name !== ''
   return (
     <React.Fragment>
-      <SettingLabel text="Difficulty multipliers" />
-
+      <SettingLabel text={`Difficulty ${fundingModel === 'Hourly' ? 'multipliers' : ''}`} />
       {expLevels.map((exp, index) => (
-        <Field
-          css={`margin-bottom: ${1.5 * GU}px`}
-          key={index}
-          label={'LEVEL ' + index}
-        >
-          <div css="display: flex">
-            <NumberInput
-              css="width: 32%; margin-right: 3%"
-              fixedDecimalScale
-              decimalScale={2}
-              value={exp.mul}
-              allowNegative={false}
-              onChange={generateExpLevelHandler(index, 'M')}
-            />
-            <TextInput
-              type="text"
-              css="font-size: 16px; width: 65%"
-              defaultValue={exp.name}
-              onChange={generateExpLevelHandler(index, 'N')}
-            />
-          </div>
-        </Field>
+        <React.Fragment key={index}>
+          {fundingModel === 'Hourly' ? (
+            <Field
+              css={`margin-bottom: ${1.5 * GU}px`}
+              label={'LEVEL ' + index}
+            >
+              <div css="display: flex">
+                <NumberInput
+                  css="width: 32%; margin-right: 3%"
+                  fixedDecimalScale
+                  decimalScale={2}
+                  value={exp.mul}
+                  allowNegative={false}
+                  onChange={generateExpLevelHandler(index, 'M')}
+                />
+                <TextInput
+                  type="text"
+                  css="font-size: 16px; width: 65%"
+                  defaultValue={exp.name}
+                  onChange={generateExpLevelHandler(index, 'N')}
+                />
+                <IconContainer
+                  theme={theme}
+                  disabled={expLevels.length <= 1}
+                  style={{ transform: 'scale(.8)' }}
+                  onClick={() => onRemoveExpLevel(index)}
+                  title="Remove this difficulty multiplier"
+                >
+                  <IconRemove />
+                </IconContainer>
+              </div>
+            </Field>
+          ) : (
+            <div
+              css={`display: flex; margin-bottom: ${1.5 * GU}px`}
+            >
+              <TextInput
+                type="text"
+                css="font-size: 16px; flex-grow: 1"
+                defaultValue={exp.name}
+                onChange={generateExpLevelHandler(index, 'N')}
+              />
+              <IconContainer
+                theme={theme}
+                disabled={expLevels.length <= 1}
+                style={{ transform: 'scale(.8)' }}
+                onClick={() => onRemoveExpLevel(index)}
+                title="Remove this difficulty multiplier"
```

Comment:
This should say "Remove this difficulty" or maybe "Remove this difficulty setting," but should not say "multiplier."